### DISCUSSION TITLE
Retrieving the original message a thread is created with

### DISCUSSION THREAD

**Discussion by cysabi** [discordjs/discord.js#8807] [Q&A] [answered]
Hello, I currently have a bot that creates threads from other people's messages automatically. The `ownerId` of the thread is the bot, not the person's original message.

Is there a way to retrieve the original message object that the thread was created from?

**Comment by MrMythicalYT** [comment, marked as answer]
https://discord.js.org/#/docs/discord.js/main/class/ThreadChannel?scrollTo=fetchStarterMessage
`ThreadChannel#fetchStarterMessage()` will return the message
